packages/openapi-generator/README.md

```diff
@@ -1,39 +1,89 @@
-## OpenApi Client Generator
+## SAP Cloud SDK OpenAPI Client Generator (Beta)
 
-This is currently not release ready. excluded in `mail.ts`.
+Generate custom JavaScript/TypeScript clients for OpenAPI services.
+This generator is based on the official [OpenAPI generator](https://openapi-generator.tech/) and adds some additional code for convenience to better integrate with the SAP Cloud SDK.
```

Comment:
I am not a big fan of the `official OpenAPI generator`. I know it is the big one but OpenApi is a open spec so I think there is no `offical` generator. But also I am a bit picky again.